pkg/rke2/rke2_linux.go

```diff
@@ -178,3 +190,19 @@ func initExecutor(clx *cli.Context, cfg Config, dataDir string, disableETCD bool
 		ControlPlaneMounts:    extraMounts,
 	}, nil
 }
+
+func getFQDNHostname() (string, error) {
+	cmd := exec.Command("hostname", "-f")
+
+	var b bytes.Buffer
+	cmd.Stdout = &b
+
+	if err := cmd.Run(); err != nil {
+		return "", err
+	}
+
+	fqdn := b.String()
+	fqdn = fqdn[:len(fqdn)-1]
+
+	return fqdn, nil
```

Comment:
```suggestion
	return strings.TrimSpace(b.String()), nil
```

Reply:
If we do this, then we don't need line 205.

Reply:
fixed